less/Forum/Item.less

```diff
@@ -167,12 +167,12 @@
       }
```

Comment:
Could you add a `>` before the `li`, making it look like: `> li`? Because the forum sidebar nav items are currently receiving a `margin-bottom` as well. (line 164)